inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java

```diff
@@ -28,15 +28,10 @@
 public class Main {
  
     public static Blob imageToBlob(Mat image) {
-        if (buff == null) 
-            buff = new byte[(int) (image.total() * image.channels())];
-
-        image.get(0, 0, buff);
-
         int[] dimsArr = {1, image.channels(), image.height(), image.width()};
         TensorDesc tDesc = new TensorDesc(Precision.U8, dimsArr, Layout.NHWC);
 
-        return new Blob(tDesc, buff);
+        return new Blob(tDesc, image.dataAddr());
```

Comment:
It's correct because we keep the source frame in `processedFramesQueue`. The frame is removed only after inference completed

Reply:
Please add a comment as a reminder about it.